_oasis.in

```diff
@@ -6,8 +6,8 @@ Authors:     Dave Scott
 License:     LGPL-2.1 with OCaml linking exception
 Plugins:     META (0.2)
 OCamlVersion: >= 4.01
-AlphaFeatures: ocamlbuild_more_args
-XOCamlbuildPluginTags: package(cppo_ocamlbuild)
+AlphaFeatures: ocamlbuild_more_args, compiled_setup_ml
+XOCamlbuildPluginTags: package(cppo_ocamlbuild), package(bisect_ppx.ocamlbuild)
```

Comment:
I just wanted to mention that it uses a different method than this PR, but I don't actually understand the details of either. One advantage of the script that I linked is that we don't need to modify the original `_oasis` file. Maybe we could just copy the relevant parts of the `instrument_oasis` function if we decide to use that method.